packages/date-picker/src/DatePicker/components/Calendar/Calendar.scss

```diff
@@ -251,3 +251,18 @@ $rdp-cell-size: 40px;
   background-color: $color-blue-100;
   color: $color-blue-500;
 }
+
+// Add Storybook psuedo classes
+:global(.story__day-hover) {
```

Comment:
Ideally, we'd be able to put this class just wherever we've got the styles already defined. This way, we wouldn't need to change multiple places to ensure they align.